src/main/scala/fr/polytechnique/cmap/cnam/study/pioglitazone/PioglitazoneMain.scala

```diff
@@ -18,7 +18,7 @@ import fr.polytechnique.cmap.cnam.etl.patients.Patient
 import fr.polytechnique.cmap.cnam.etl.sources.Sources
 import fr.polytechnique.cmap.cnam.etl.transformers.exposures.ExposuresTransformer
 import fr.polytechnique.cmap.cnam.etl.transformers.follow_up.FollowUpTransformer
-import fr.polytechnique.cmap.cnam.etl.transformers.follow_up.FollowUpTransformer.FollowUpDataset
+import fr.polytechnique.cmap.cnam.etl.transformers.follow_up.FollowUpTransformerUtilities.FollowUpDataset
```

Comment:
This is bad. Utilities shoudn't be involved in high level classes like this one.

Reply:
ok